app/scripts/router.js

```diff
@@ -284,6 +284,8 @@ function (
         });
     },
 
+    showViewGenerator: showViewGenerator,
```

Comment:
@zaach  Should we add this or have a private ref: `_showViewGenerator` just for testing purposes?